app/src/common/chromium/com/igalia/wolvic/browser/api/impl/DisplayImpl.java

```diff
@@ -46,13 +46,6 @@ public void surfaceChanged(@NonNull Surface surface, int width, int height) {
 
             // Dispatch onSurfaceChanged
             mRenderView.surfaceChanged(surface, width, height);
-
-            postDelayed((Runnable) () -> {
-                @Nullable WSession.ContentDelegate delegate = mSession.getContentDelegate();
-                if (delegate != null) {
-                    delegate.onFirstComposite(mSession);
-                }
-            }, 100);
```

Comment:
Why was this removed?

Reply:
FYI, this branch is based on `experimental/content_api_backend`(https://github.com/Igalia/wolvic/tree/experimental/content_api_backend) branch worked by Zarhar. So, I think he could give us more of the context of this code, but I feel that this code is unnecessary since `mAfterFirstPaint` in WindowWidget.onFirstComposite that is |delegate| will still be `false` after 100 ms,  because this flag needs more time to change to `true`  for the first paint of web page.

Reply:
This bit was originally written from the weblayer implementation so it must be outdated. Explanation above makes sense.

Reply:
I think that we can remove reference of mSession in DisplayImpl as well, so that  the circular dependency between DisplayImpl and Session Impl can be removed.

Reply:
@duswnchl could you clean up SessionImpl's dependency when you work on the Tab support? Considering the comment below, I thought that this code would be used later when supporting the Tab. If not, we could remove it.
`
       // Make sure we update the current active tab. We have an empty SessionImpl.setActive implementation
        // because in weblayer the browser window is selected from a pool. We can use the hint that
        // the session want's to be rendered into this surface to set it active too.
`

Reply:
@mshin-wolvic Aha, Sure. I'll consider it on Tab support. Thanks!